doc/fortunes.tips

```diff
@@ -38,7 +38,7 @@ Step through your seek history with the commands 'u' (undo) and 'U' (redo)
 Use hasher to calculate hashes of portion blocks of a file
 Use zoom.byte=entropy and press 'z' in visual mode to zoom out to see the entropy of the whole file
 Use 'zoom.byte=printable' in zoom mode ('z' in Visual mode) to find strings
-Set color to your screen with 'e scr.color=true'
+Set color to your screen with 'e scr.color=1' or truecolor with 'e scr.color=3'
```

Comment:
"Add color to your screen with 'e scr.color=X' where 1 is 16 colors, 2 is 256 colors and 3 is 16M colors'